applications/FemToDemApplication/python_scripts/MainFEM_for_coupling.py

```diff
@@ -31,6 +32,59 @@ def Initialize(self):
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NORMAL)
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.PRESSURE)
 
+        # Adding PFEM Variables TODO put in another place

```

Comment:
Pfem uses a lot of historical values, this is super expensive in memory, pfem group should change that

Reply:
I know... I ping @AFranci just to keep him informed

Reply:
Hi @loumalouomega, do you some suggestions to make it better? thanks...